tests/sentry/backup/test_releases.py

```diff
@@ -91,220 +101,104 @@ def mark(*marking: Type | Literal["__all__"]):
-        incident = self.create_incident()
-        IncidentActivity.objects.create(
-            incident=incident,
-            type=1,
-            comment="hello",
-        )
-        IncidentSnapshot.objects.create(
-            incident=incident,
-            event_stats_snapshot=TimeSeriesSnapshot.objects.create(
-                start=datetime.utcnow() - timedelta(hours=24),
-                end=datetime.utcnow(),
-                values=[[1.0, 2.0, 3.0], [1.5, 2.5, 3.5]],
-                period=1,
-            ),
-            unique_users=1,
-            total_events=1,
-        )
-        IncidentSubscription.objects.create(incident=incident, user_id=user.id)
-        IncidentTrigger.objects.create(
-            incident=incident,
-            alert_rule_trigger=trigger,
-            status=1,
-        )
-
-        # *Snapshot
-        PendingIncidentSnapshot.objects.create(
-            incident=incident, target_run_date=datetime.utcnow() + timedelta(hours=4)
-        )
-        TimeSeriesSnapshot.objects.create(
-            start=datetime.utcnow() - timedelta(hours=24),
-            end=datetime.utcnow(),
-            values=[[1.0, 2.0, 3.0], [1.5, 2.5, 3.5]],
-            period=1,
-        )
-
-        # Relay*
-        _, public_key = generate_key_pair()
-        relay = str(uuid4())
-        Relay.objects.create(relay_id=relay, public_key=str(public_key), is_internal=True)
-        RelayUsage.objects.create(relay_id=relay, version="0.0.1", public_key=public_key)
-
-        # Dashboard
-        dashboard = Dashboard.objects.create(
-            title="Dashboard 1", created_by_id=user.id, organization=org
-        )
-        widget = DashboardWidget.objects.create(
-            dashboard=dashboard,
-            order=1,
-            title="Test Widget",
-            display_type=0,
-            widget_type=DashboardWidgetTypes.DISCOVER,
-        )
-        DashboardWidgetQuery.objects.create(widget=widget, order=1, name="Test Query")
-        DashboardTombstone.objects.create(organization=org, slug="test-tombstone")
-
-        # *Search
-        RecentSearch.objects.create(
-            organization=org,
-            user_id=user.id,
-            type=SearchType.ISSUE.value,
-            query="some query",
-        )
-        SavedSearch.objects.create(
-            organization=org,
-            name="Saved query",
-            query="saved query",
-            visibility=Visibility.ORGANIZATION,
-        )
-
-        # misc
-        Counter.increment(project, 1)
-        Email.objects.create(email="other@example.com")
-        self.create_notification_action(organization=org, projects=[project])
-        Repository.objects.create(
-            name="test_repo",
-            organization_id=org.id,
-            integration_id=self.integration.id,
-        )
-
-        return self.import_export_then_validate()
+        return ReleaseFixtureGenerator()(self._testMethodName)
+
+    @targets(
+        mark(
+            Actor,
```

Comment:
what's the reason of not using `__all__` here?

Reply:
Oh, it's because the models here change over time. Got it. Can this be stored somewhere else? This could get super large over time

Reply:
I'll leave a not to improve this, though I'm not sure that moving it is best. I agree that its a bit ugly, but it's hard to think of a better place for it to live then next to the test, and it will be large no matter where we put it.